js/ui/apple2.js

```diff
@@ -558,9 +558,9 @@ function _keydown(evt) {
         keyboard.commandKey(true);
     } else if (evt.keyCode == 18) { // Alt
         if (evt.location == 1) {
-            keyboard.commandKey(true);
-        } else {
             keyboard.optionKey(true);
+        } else {
+            keyboard.commandKey(true);
```

Comment:
This configuration was selected to allow keyboards with only Alt keys (in this case a Chromebook) to be able to use the apple keys and match the layout. I'm not sure this change matches the correct behavior, since now neither right side option or command activates closed apple.

Reply:
The change was so my Mac keyboard would do both open- and closed- apple on option and command keys. Apologies if it breaks other keyboards/OSs.

Reply:
The way it currently works is both command/open-apple keys generate open-apple, and left option/alt generates open-apple and right alt generates closed-apple, which I think is the standard Mac keyboard. The problem is probably more of a doc issue. Someday I will write some docs.

Reply:
Sorry. My previous comment was wrong. The change was because the left option key on the Mac keyboard was triggering the right (closed-apple) key on the virtual keyboard, and the right option key on the Mac keyboard was triggering the left (open-apple) key on the virtual keyboard. With the change above, the side of the Mac keyboard key matched the virtual keyboard. I also have my Mac option and command keys swapped, so for me it was the command keys in each case. However, it is not a big issue. Let me know if you want any more help with this.